app/models/RegulatorsAndDocuments.scala

```diff
@@ -28,22 +28,32 @@ object CharityCommissionRegistrationNumber {
 
 }
 
-case class CharityOtherRegulatorDetails(regulatorName:String, registrationNumber:String)
+case class ScottishRegulatorRegNumber(registrationNumber:String)
 
-object CharityOtherRegulatorDetails {
+object ScottishRegulatorRegNumber {
 
-  implicit val formats = Json.format[CharityOtherRegulatorDetails]
+  implicit val formats = Json.format[ScottishRegulatorRegNumber]
 
-  override def toString: String = "charityOtherRegulatorDetails"
+  override def toString: String = "scottishRegulatorRegNumberDetails"
 
 }
 
-case class ScottishRegulatorRegNumber(registrationNumber:String)
+case class NIRegulatorRegNumber(nIRegistrationNumber: String)
 
-object ScottishRegulatorRegNumber {
+object NIRegulatorRegNumber{
 
-  implicit val formats = Json.format[ScottishRegulatorRegNumber]
+  implicit val formats = Json.format[NIRegulatorRegNumber]
 
-  override def toString: String = "scottishRegulatorRegNumberDetails"
+
+  override def toString: String = "NIRegulatorRegNumberDetails"
```

Comment:
Please modify the value like starts with lowercase.... because we follow this standard everywhere